src/style.css

```diff
@@ -1,3 +1,78 @@
-body {
+:root {
+  --maincolor: #00acdc;
+  --header-height: 60px;
+  --header-margin: 50px;
+}
+
+* {
   margin: 0;
+  padding: 0;
+}
+
+body {
+  font-family: Roboto, sans-serif;
+}
+
+.header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+
+  .logo {
+    display: flex;
+    align-items: center;
+    margin-left: var(--header-margin);
+  }
+
+  .nav {
+    margin: 0 var(--header-margin) 0 0;
+  }
```

Comment:
CSS rules for `.nav` should not be nested inside `.header`. Ensure that your CSS follows the standard rules without nesting unless you are using a preprocessor that supports it.

Reply:
new css